### ISSUE TITLE
JS: Symbols in command palette are truncated.

### ISSUE THREAD

**Issue by phindle** [Microsoft/vscode#8994]
- VSCode Version: Code 1.3.0 (e724f269ded347b49fcf1657fc576399354e6703, 2016-07-07T16:56:12.476Z)
- OS Version: Windows_NT ia32 10.0.10586

Steps to Reproduce:

1. I have the simple script (pasted below) open in VS Code.
2. Press Ctrl+P to invoke command palette
3. Type the @ symbol to display symbols in the script.
4. (Refer to screenshot) Symbols are truncated with ... Unable to see the full symbol name.

Notes : Running on a 4K display, full-screen.

```
var CapsuleSpinner = pc.createScript('capsuleSpinner');

CapsuleSpinner.attributes.add('spinSpeed', {
    type: 'number',
    default: 12
});

CapsuleSpinner.prototype.initialize = function() {
    this.angle = 30;
};

CapsuleSpinner.prototype.update = function(dt) {
    this.angle += this.spinSpeed * dt;
    this.entity.setEulerAngles(this.angle, 0, 0);
};

CapsuleSpinner.prototype.swap = function(old) {
    
};
```

![image](https://cloud.githubusercontent.com/assets/4445791/16707087/ffd588ea-45bb-11e6-8450-95b9e10d746b.png)

**Comment by bpasero** [comment]
@phindle could you quickly bring up the chrome debug tools (via help menu) and see if literally the text comes truncated or if the widget does the truncation?

**Comment by phindle** [comment]
Hi Benjamin. Upon viewing the Quick-Open widget in the element inspector, I can see the text appears to be truncated. I've attached another screenshot which shows the situation in the debug tools.

Many thanks
Phil

![vscode-extensions-debugger](https://cloud.githubusercontent.com/assets/4445791/16725633/11b4f5a6-4750-11e6-8d6a-3ebd51db2131.png)

**Comment by bpasero** [comment]
@phindle is this JavaScript? Do you have any extensions installed?

**Comment by phindle** [comment]
Yes, this is JavaScript.

I have the following extensions installed:

Bookmarks (0.8.0)
ESLint (0.10.18)
Debugger for Chrome (0.4.7)
Son of Obsidian Theme (0.2.0)
Git Blame (1.1.0)

**Comment by bpasero** [comment]
Might be Salsa then? We are not trimming any strings in the UI and from the screenshot it looks as if the string is already truncated when it returns from the language service.

**Comment by phindle** [comment]
It's possible. I don't know enough about Salsa, or the internals of VSCode to offer much.
Am I able to debug Salsa? Is it part of the VS Code main repo?

**Comment by dbaeumer** [comment]
This is Salsa and fixed in 2.0. See https://github.com/Microsoft/TypeScript/issues/8238
